Desktop/Application/MaxMix/Services/Audio/AudioSession.cs

```diff
@@ -116,13 +116,28 @@ public bool IsMuted
         private void UpdateDisplayName()
         {
             var displayName = _session2.DisplayName;
-            if (IsSystemSound) { displayName = "System Sounds"; }
-            if (string.IsNullOrEmpty(displayName)) { displayName = _session2.Process.MainWindowTitle; }
-            if (string.IsNullOrEmpty(displayName)) { displayName = _session2.Process.GetProductName(); }
-            if (string.IsNullOrEmpty(displayName)) { displayName = _session2.Process.ProcessName; }
-            if (string.IsNullOrEmpty(displayName)) { displayName = "Unnamed"; }
-            displayName = char.ToUpper(displayName[0]) + displayName.Substring(1);
+            if (IsSystemSound) {
+                displayName = "System Sounds";
+            }
+            else
+            {
+                var productName = _session2.Process.GetProductName();
+                if (productName == "Spotify")
```

Comment:
Don't special case a single application like this. Lets just prioritize GetProductName over MainWindowTitle